api/src/test/java/org/openmrs/module/metadatadeploy/handler/impl/ConceptDeployHandlerTest.java

```diff
@@ -101,6 +101,7 @@ public void testCreateNumeric() throws Exception {
         numeric.addDescription(new ConceptDescription("1-10", Locale.ENGLISH));
         numeric.setLowAbsolute(1d);
         numeric.setHiAbsolute(10d);
+        //The method setPrecise(boolean) is undefined for the type ConceptNumeric
```

Comment:
Because it was removed at https://github.com/openmrs/openmrs-core/commit/d65825b685453515f55e30504e9cd92ebaa3f759#diff-c6944a72a4cb885dba94e8b377659090 prior openmrCore 2.1.0

Reply:
Is the above comment really required?